python-backend/main.py

```diff
@@ -66,16 +69,23 @@ async def analyze(request: Request):
 
     try:
         data = Input(**(await request.json()))
-        (expression, numerator, denominator, symbol) = parse(data)
-        pcf = ramanujan.pcf.PCF(str(numerator), str(denominator))
-        limit = pcf.limit(depth=data.i)
-        computed_value = LIReC.db.access.identify(limit)
+        (expression, a, b, symbol) = parse(data)
+        pcf = ramanujan.pcf.PCF(str(a), str(b))
+        limit = mpmath.mpf(pcf.limit(depth=data.i))
+        logger.debug(f"PCF limit() returned: {limit}")
+        computed_value = db.identify(values=[limit], wide_search=True)
```

Comment:
nit: should this be `computed_values` since they are being enumerated on the next line?

Reply:
it was originally one value and I simply overlooked it when I switched the invocation to the correct `identify`